bucket/v2rayn.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "2.25",
+    "description": "GUI client based on v2ray core",
+    "homepage": "https://github.com/2dust/v2rayN",
+    "license": "GPL-3.0",
+    "url": "https://github.com/2dust/v2rayN/releases/download/2.25/v2rayN.zip",
+    "hash": "fa915d5fec22cad85ea65edfd920c30340d04a64ead52fe718161978b2242c90",
+    "extract_dir": "v2rayN",
+    "bin": "v2rayN.exe",
+    "persist": [
+        "user-wininet.json",
+        "config.json"
+    ],
+    "depends": "v2ray",
+    "shortcuts": [
+        [
+            "v2rayN.exe",
+            "v2rayN"
+        ]
+    ],
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\user-wininet.json\")) {",
+        "    New-Item -Force -Path \"$persist_dir\" -Name 'user-wininet.json' -Value '{}' | Out-Null",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\config.json\")) {",
+        "    Copy-Item \"$(appdir v2ray)\\current\\config.json\" $persist_dir | Out-Null",
+        "}",
+        "'*.exe', '*.dat', 'vpoint*.json' | % { % ($file in dir \"$(appdir v2ray)\\current\") {",
```

Comment:
This syntax is not readable at all. 

What is $_ in this context?
Convert it to something like:

```powershell
foreach $form in @(..formats) {
    current_dir \"$dir\" | ForEach-Object {
        if $_ -like $form { .....
    }
}
```